config/initializers/assets.rb

```diff
@@ -5,6 +5,7 @@
 
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
Can you note in the README how you even Node? I have no idea personally how I'd update these. Is it possible to pull them in with a Gem instead? NP if not.